main.js

```diff
@@ -44,6 +44,26 @@ const checkSafeFrame = () => {
 	}
 };
 
+const monitorHeavyAds = () => {
+	// create the observer with the callback
+	const observer = new window.ReportingObserver(
```

Comment:
This will need to be behind a feature detection because Reporting Observer is only in Chromium browsers -- https://caniuse.com/mdn-api_reportingobserver

Reply:
Argh! Nice catch @JakeChampion! I'll add it.